runtime/runtime/src/lib.rs

```diff
@@ -1165,6 +1167,11 @@ impl Runtime {
 
         let trie = Rc::new(trie);
         let mut state_update = TrieUpdate::new(trie.clone());
+        let mut prefetcher = TriePrefetcher::new(trie.clone());
```

Comment:
I don't understand, why do you recreate a prefetcher from scratch, including spinning up new threads, for each new block? Would it not be possible to just reuse the same thread pool and metadata for all the blocks, just clearing out the slots to make space for the new prefetch requests?

Reply:
Hm, it would be nicer to keep the prefetcher and threads alive, yeah. The trie root of each IO thread's `PrefetchingTrieStorage` would have to be updated, as each chunk operates on a different trie. It can also be tricky because chunks are already executed by a rayon iterator, so we don't even know exactly how many chunks are being processed at a time.
And we would need to keep state between chunks somewhere and pass it down. In other words, the change would spill much further up in the transaction runtime, while I was trying to keep it local.

How bad do you think it is to create a new set of threads each time? If it is only a problem due to performance concerns, it might be okay to current design, my tests showed that creating the threads only costs single digit micro seconds.